server/controllers/questions.js

```diff
@@ -0,0 +1,58 @@
+import knexModule from 'knex';
+import bookshelfModule from 'bookshelf';
+import questionsService from '../db/services/questions';
+import { development as devconfig } from '../../knexfile';
```

Comment:
I think this has been resolved. We can talk about this more indepth as a group though